src/modules/userCreationRequestMethods.ts

```diff
@@ -611,10 +701,36 @@ export const userCreationRequestsMethods = {
     getList: async (data: GetUserCreationRequestList): Promise<CompleteUserCreationRequest[]> => {
         const where: Prisma.UserCreationRequestWhereInput = {};
 
+        if (data.type) {
+            where.type = { in: data.type };
+        }
+
+        if (data.status !== undefined) {
```

Comment:
`status !== undefined` ->  `where.status = null` looks suspicious

Reply:
we have two statuses: Accepted and Declined, if request on consideration, its status === null. By default we need to show only requests with nullish status